packages/sanity/src/core/form/inputs/PortableText/text/ListItem.tsx

```diff
@@ -0,0 +1,26 @@
+import {BlockListItemRenderProps} from '@sanity/portable-text-editor'
+import React, {useCallback, useMemo} from 'react'
+
+export const ListItem = (props: BlockListItemRenderProps) => {
+  const {block, type, selected, focused, level, value, renderDefault} = props
+  const children = useMemo(() => renderDefault(props), [props, renderDefault])
```

Comment:
This will be changed through this later on: https://github.com/sanity-io/sanity/pull/3971

We will remove the `renderDefault` here because it's nothing you can change here with props anyway. Instead we just pass `children` as normal.